packages/suite/src/components/wallet/PrivacyAccount/CoinjoinSetup.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { AnimatePresence, motion } from 'framer-motion';
+import styled from 'styled-components';
+
+import { Translation } from '@suite-components';
+import { useSelector } from '@suite-hooks/useSelector';
+import { Card, RadioButton, motionAnimation, Warning } from '@trezor/components';
+import { coinjoinAccountUpdateSetupOption } from '@wallet-actions/coinjoinAccountActions';
+import { AnonymityLevelSetup } from '@wallet-components/PrivacyAccount/AnonymityLevelSetup';
+import { MaxMiningFeeSetup } from '@wallet-components/PrivacyAccount/MaxMiningFeeSetup';
+import { SkipRoundsSetup } from './SkipRoundsSetup';
+import { selectCoinjoinAccountByKey } from '@wallet-reducers/coinjoinReducer';
+
+const SetupContainer = styled.div`
+    padding: 18px;
+`;
+
+const Wrapper = styled.div`
+    display: flex;
+    flex-direction: column;
```

Comment:
I know it's vertical in Figma but what about going with this at least, @hynek-jina? Less space is wasted. 

<img width="500" alt="image" src="https://user-images.githubusercontent.com/45338719/224403615-d4082f05-96e5-4b25-87cb-aae83a99b948.png">

Reply:
I would rather stick to vertical sort since it is general pattern... But not a strong preference.

Reply:
@hynek-jina The amount of negative space created by the vertical alignment makes me sad 😞

Reply:
Let's do it that way then.. Ondra is not against it..

Just small note from him. 
Text should be closer to radio button so the relation is obvious.. and the space between elements bigger on the other hand..

So the top one is bad and the bottom one is correct 🙂 
<img width="752" alt="14-march-10-30_2x" src="https://user-images.githubusercontent.com/26002916/224976329-ac23c664-d1af-41ec-99be-0a2bc06ee22c.png">

Reply:
c922eaf56d5963cbe9b5feb21f85cd2d40b12bb1

![Screenshot 2023-03-14 at 12 15 56](https://user-images.githubusercontent.com/42465546/224985028-79c98c20-7d30-43bb-b757-a5908eaed827.png)

I decreased the `label` padding globally as I think it was a bit too high.